backend/CMakeLists.txt

```diff
@@ -16,3 +19,20 @@ add_executable(${PROJECT_NAME} ${SOURCES})
 target_link_libraries(${PROJECT_NAME} Crow::Crow SQLite::SQLite3)
 
 target_include_directories(${PROJECT_NAME} PRIVATE src)
+
+## tests
+project(tests)
+
+find_package(Catch2 REQUIRED)
+
+set (TEST_SOURCE
+    tests/test.cpp
+)
+
+add_executable(${PROJECT_NAME} ${TEST_SOURCE})
+
+target_link_libraries(${PROJECT_NAME} PRIVATE Catch2::Catch2WithMain)
+
+target_include_directories(${PROJECT_NAME} PRIVATE tests)
+
+include(Catch)
```

Comment:
**question**: Why is this here?

Reply:
In the documents for Catch they include this line but do not include the direct reason: 

[Catch Github documents](https://github.com/catchorg/Catch2/blob/devel/docs/cmake-integration.md#top)

Reply:
I am going to delete it though since it is not impacting the running of the backend folder.